src/Meta/Metadata.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\BlockBundle\Meta;
+
+/**
+ * @author Hugo Briand <briand@ekino.com>
+ */
+class Metadata implements MetadataInterface
```

Comment:
should this extend the core one? What if someone is depending on the actual concrete class.

Reply:
General speaking I'm with you, but we don't replace any service and the user can decide which implementation (CoreBundle oder BlockBundle) of `MetadataInterface` we wants.